# Generated by Django 5.1.1 on 2024-09-24 13:03

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('appproject', '0005_fieldselfrelationship_unique_field_subfield'),
    ]

    operations = [
        migrations.RemoveConstraint(
            model_name='fieldselfrelationship',
            name='unique_field_subfield',
        ),
        migrations.AddConstraint(
            model_name='fieldselfrelationship',
            constraint=models.UniqueConstraint(fields=('field', 'subfield'), name='unique_field_subfield', violation_error_message='This relationship already exists.'),
        ),
    ]
